src/Admin/IdeasWorkshopIdeaAdmin.php

```diff
@@ -189,11 +189,8 @@ protected function configureListFields(ListMapper $listMapper)
                     'show_contributors' => [
                         'template' => 'admin/ideas_workshop/idea/list_show_contributors.html.twig',
                     ],
-                    'enabled' => [
-                        'template' => 'admin/ideas_workshop/idea/list_action_enabled.html.twig',
-                    ],
-                    'disabled' => [
-                        'template' => 'admin/ideas_workshop/idea/list_action_disabled.html.twig',
+                    'moderate' => [
+                        'template' => 'admin/ideas_workshop/thread/list_action_moderate.html.twig',
```

Comment:
maybe move this template in the `admin/ideas_workshop/` directory?

Reply:
Done.